Cargo.toml

```diff
@@ -1,10 +1,12 @@
 [package]
 name = "fdlimit"
-version = "0.2.1"
+version = "0.3.0"
```

Comment:
This is a breaking change to the API, should it be a major bump?

Reply:
It isn't at `1.x` yet, so minor bump is appropriate here. Unless everyone is happy to go to 1.0 with this PR.

Reply:
API breakage is only a minor version bump on < 1.0.0